docs/editors/vim.md

```diff
@@ -63,10 +63,11 @@ Language Server Protocol support to Vim/Nvim  to communicate with the Metals
 language server.
 
 Assuming [`vim-plug`](https://github.com/junegunn/vim-plug) is used (another
-plugin manager like vundle works too), update your `~/.vimrc` to include the
+plugin manager like vundle works too, but requires slightly different configuration), update your `~/.vimrc` (or, in case of Neovim, `~/.config/nvim/init.vim`) to include the
 following settings.
 
 ```vim
+call plug#begin()
```

Comment:
I think I'd leave this out to potentially avoid people copy pasting this in and having multiple calls to `plug#begin()`. There is a bit of assumed knowledge here about how a plugin manager works. The reason I only include the `Plug` line is that most plugins that list installation instructions do it this way, so it should be pretty standard.

Reply:
That makes sense. I was thinking of someone using vim-plug for the first time, encountering an error after copy pasting only the Plug line - but it can of course be expected from people to read the vim-plug README first.